src/cobra/util/array.py

```diff
@@ -112,67 +126,81 @@ def nullspace(A, atol=1e-13, rtol=0):
     return ns
 
 
-def constraint_matrices(model, array_type="dense", include_vars=False, zero_tol=1e-6):
+def constraint_matrices(
```

Comment:
I wonder if this function is still in use anywhere or by anyone? Would be nice to remove it.

Reply:
This is only there for non-optlang based use cases. I think it's better to issue a deprecation for now and remove it completely in the major version upgrade.

Reply:
Yes, that's a good solution if we indeed want that. At the moment I simply don't know if the format is still in use at all.

Reply:
Since we are moving towards a leaner code base, I think it's better if we drop it. Maybe we could keep it in the cookbook?!

Reply:
Let's go with the deprecation warning for now.

Reply:
Okay.

Reply:
Those are used in the sampler, please don't remove them. Actually, most of the methods here are required for sampling.

Reply:
I see. I won't issue a deprecation for the function then.